SPECS/grub2/50_mariner

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+cat << EOF
+set bootprefix=$GRUB_MARINER_BOOTPREFIX
+load_env -f $GRUB_MARINER_BOOTPREFIX/mariner.cfg
+EOF
```

Comment:
Out of curiosity, is it required these be split into multiple `cat` commands?

Reply:
Fixed.